cognite/neat/_session/_to.py

```diff
@@ -35,5 +37,30 @@ def yaml(self, io: Any | None = None) -> str | None:
 
 
 class CDFToAPI:
-    def __init__(self, client: CogniteClient | None) -> None:
+    def __init__(self, state: SessionState, client: CogniteClient | None, verbose: bool) -> None:
         self._client = client
+        self._state = state
+        self._verbose = verbose
+
+    def instances(self, space: str | None = None):
+        if not self._state.verifies_dms_rules:
+            raise ValueError("No verified DMS data model available")
+
+        space = space or self._state.verifies_dms_rules.metadata.space
+        loader = loaders.DMSLoader.from_rules(self._state.verifies_dms_rules, self._state.store, space)
+
+        if self._client:
+            return loader.load_into_cdf(self._client)
+        else:
+            return loader.write_to_file(Path("./instance.yaml"))
```

Comment:
Isn't this confusing?  Shouldn't we instead have that placed somewhere else
```
neat.to.file.yaml(format="dms")
```

It is a bit unintuitive to me that if 
```
neat.to.cdf...()
```
I expect to push to CDF.

Reply:
Agree